src/main/java/pl/vm/javaguild/designpatterns/pattern/creational/factorymethod/Department/Department.java

```diff
@@ -0,0 +1,30 @@
+package pl.vm.javaguild.designpatterns.pattern.creational.factorymethod.Department;
+
+import pl.vm.javaguild.designpatterns.pattern.creational.factorymethod.Employee.Employee;
```

Comment:
Problem z pakietami. Proponuję przeniesienie modelu danych do pakietu `model`

```suggestion
import pl.vm.javaguild.designpatterns.pattern.creational.factorymethod.model;
```